src/staking-router-modules/contracts/lido-locator/lido-locator.service.ts

```diff
@@ -13,8 +13,14 @@ export class LidoLocatorService {
   ) {}
 
   async getStakingRouter(blockTag: BlockTag) {
-    // if set address via attach this.contract.address will still have the same address
-    this.loggerService.log('Contract locator address', this.contract.address);
+    const locatorENVAddress = this.config.get('LIDO_LOCATOR_ADDRESS');
+
+    this.loggerService.log('Contract locator address', { contract: this.contract.address, env: locatorENVAddress });
```

Comment:
Let's make the names of the contract and environment more descriptive to understand why we have different addresses. Both of them are contracts.